client/performance-profiler/pages/dashboard/index.tsx

```diff
@@ -28,10 +39,18 @@ export const PerformanceProfilerDashboard = ( props: PerformanceProfilerDashboar
 					url={ url }
 					active={ activeTab }
 					onTabChange={ getOnTabChange }
+					isLoading={ ! desktopLoaded }
```

Comment:
On https://github.com/Automattic/wp-calypso/pull/93468 I've added a flag called `showNavigationTabs` that can be used here.

Reply:
It seems to me that navigation tabs are shown always, even while loading. I have initially added the isLoading boolean to hide the `Date` from the header.

Reply:
> It seems to me that navigation tabs are shown always

I see now, I was not aware of this. In any case maybe we should add a boolean with a similar name, like `showLastUpdatedDate` instead of using `isLoading` as this component shouldn't handle a lot of business logic inside of it.